pkg/polaris/ctl_polaris.go

```diff
@@ -122,16 +122,16 @@ func (ctl *CRSpecBuilderFromCobraFlags) SetPredefinedCRSpec(specType string) err
 func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Command, master bool) {
 
 	// [DEV NOTE:] please organize flags in order of importance
-	cmd.Flags().StringVar(&ctl.Version, "version", ctl.Version, "Version of Polaris\n")
+	cmd.Flags().StringVar(&ctl.Version, "version", ctl.Version, "Version of Polaris")
```

Comment:
can we keep the new lines?  These are meant to act as grouping for now 😄

Reply:
Done!